atlp-rwanda/trojans-ec-fe
# #184760093 ft Users update their password

what does the PR do?

PR to enable users to update their password

Description of the task completed
the task completed is to enable users to update their password upon providing their old and new password

How can it be manually tested
it can be manually tested by logging in and navigating to the dashboard to update the password, then provide old, new, and confirm password

Any background context you want to provide
N/N

What are the relevant pivotal tracker stories?

Users should be able to update their password

what is your pivotal tracker link ?
https://www.pivotaltracker.com/n/projects/2628461)

![passwordupdate](https://user-images.githubusercontent.com/88782915/235689982-549bb9a6-dd86-4793-b3f5-91f5876f983a.png)
